training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
```

Comment:
<p>I recommend you to mark which hyperparams are added for SMP config. Maybe L16-L19? </p><p>Also add reference to https://sagemaker.readthedocs.io/en/stable/api/training/smd_model_parallel_general.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Updated